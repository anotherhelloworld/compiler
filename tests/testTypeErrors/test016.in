var a, b : real;
begin
    a := 100 and b;
end.