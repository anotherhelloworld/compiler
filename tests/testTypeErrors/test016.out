Incompatible types: expected: "real" but "BadType" found Line 3 Column 19
